lib/msal-node/src/index.ts

```diff
@@ -47,6 +47,8 @@ export {
     SerializedAppMetadataEntity,
     SerializedRefreshTokenEntity,
 } from "./cache/serializer/SerializerTypes";
+export { Serializer } from "./cache/serializer/Serializer";
```

Comment:
@Sameerag Are these things reasonable to export on the public surface or do we need to reserve the right to change API signatures? If so, perhaps we should export them under 'internals', similar to msal-browser, or find a way not to rely on them in E2E. Thoughts?

Reply:
Let me export them under internals then.

Reply:
done